src/main/java/com/es/phoneshop/model/product/ProductNotFoundExeption.java

```diff
@@ -0,0 +1,4 @@
+package com.es.phoneshop.model.product;
+
+public class ProductNotFoundExeption extends Exception{
```

Comment:
Implement at least constructor with info message param. When you throw exception with message it much easier to determine the reason with info message.